website/react-magma-docs/src/pages/api/checkboxes.mdx

```diff
@@ -80,77 +80,6 @@ export function Example() {
 }
 ```
 
-## Indeterminate
```

Comment:
Because we're removing this section of the docs on this page, I think we should have a link to to the Indeterminate Checkbox docs page on this page.  Can you add a new sentence with a link to IndeterminateCheckbox  up at the top near where we also link to the radio buttons etc?